docs/src/main/docs/guides/12_tracing_se_guide.adoc

```diff
@@ -0,0 +1,785 @@
+if you trace a service endpoint that calls another service, then the trace would contain spans from both services.
+Within a trace, spans are organized as a directed acyclic graph (DAG) and
+can belong to multiple services, running on multiple hosts.  The _OpenTracing Data Model_ describes the details
+at https://opentracing.io/specification[The OpenTracing Semantic Specification].
+Spans are automatically created by Helidon as needed during execution of the REST request.
+
+== Getting started with tracing
+
+The examples in this guide demonstrate how to integrate tracing with helidon, how to view traces, how to trace
+across multiple services, and how to integrate with tracing with Kubernetes.  All examples use Zipkin and traces
+will be viewed using both the Zipkin API and UI.
+
+=== Create a sample Helidon SE project
+
+Use the Helidon SE Maven archetype to create a simple project that can be used for the examples in this guide.
+
+[source,bash,subs="attributes+"]
+.Run the Maven archetype:
+----
+mvn archetype:generate -DinteractiveMode=false \
+    -DarchetypeGroupId=io.helidon.archetypes \
+    -DarchetypeArtifactId=helidon-quickstart-se \
+    -DarchetypeVersion={helidon-version} \
+    -DgroupId=io.helidon.examples \
+    -DartifactId=helidon-quickstart-se \
+    -Dpackage=io.helidon.examples.quickstart.se
+----
+
+[source,bash]
+.The project will be built and run from the `helidon-quickstart-se` directory:
+----
+cd helidon-quickstart-se
+----
+
+=== Setup Zipkin
+
+First, you need to run the Zipkin tracer.  Helidon will communicate with this tracer at runtime.
+
+[source,bash]
+.Run Zipkin within a docker container, then check the Zipkin server health:
+----
+docker run -d --name zipkin -p 9411:9411 openzipkin/zipkin  // <1>
+----
+<1> Run the Zipkin docker image named `openzipkin/zipkin`.
+
+
+[source,bash]
+.Check the Zipkin server health:
+----
+curl http://localhost:9411/health // <1>
+...
+{
+  "status": "UP", <2>
+  "zipkin": {
+    "status": "UP",
+    "details": {
+      "InMemoryStorage{}": {
+        "status": "UP"
+      }
+    }
+  }
+}
+----
+<1> Invoke the Zipkin REST API to check the Zipkin server health.
+<2> All `status` fields should be `UP`.
+
+=== Enable Tracing in your Helidon application
+
+Update the pom.xml file and add the following Zipkin dependency to the `<dependencies>`
+section (*not* `<dependencyManagement>`).
+
+[source,xml]
+.Add the following dependency to `pom.xml`:
+----
+<dependency>
+    <groupId>io.helidon.tracing</groupId>
+    <artifactId>helidon-tracing</artifactId>
+</dependency>
+<dependency>
+    <groupId>io.helidon.tracing</groupId>
+    <artifactId>helidon-tracing-zipkin</artifactId>
+</dependency>
+----
+
+All spans sent by Helidon to Zipkin need to be associated with a service.  Specify the service name below.
+
+[source,bash]
+.Add the following line to `resources/application.yaml`:
```

Comment:
fixed see above:
"Update the pom.xml file and add the following Zipkin dependency to the `<dependencies>`
section (*not* `<dependencyManagement>`).  This will enable Helidon to use Zipkin at the
default host and port, `localhost:9411`."